snap/snapcraft.yaml

```diff
@@ -201,6 +203,8 @@ parts:
       - libgl1
       - libglib2.0-0
       - libglib2.0-dev
+      - libgstreamer1.0-dev
+      - libgstreamer-plugins-base1.0-dev
```

Comment:
libglib2.0-dev above is probably a special exception because we need some tools at runtime but i believe we'd be fine with `libgstreamer1.0-0` and `libgstreamer-plugins-base1.0-0` here because we don't need the development headers etc.